hs_core/management/commands/fix_resourcefile_duplicates.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+"""
+Fix resource file duplicates
+
+This checks that there is only one ResourceFile for each iRods file
+
+* By default, prints errors on stdout.
+* Optional argument --log instead logs output to system log.
+"""
+
+from django.core.management.base import BaseCommand
+from django.db.models import Count
+from hs_core.models import ResourceFile
+
+
+class Command(BaseCommand):
+    help = "Fix ResourceFile duplicates."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--dryrun',
+            action='store_true',
+            dest='dryrun',
+            default=False,
+            help='Find duplicates but dont fix them',
+        )
+
+    def handle(self, *args, **options):
+        dry_run = options['dryrun']
+
+        dup_resource_files = ResourceFile.objects.values('resource_file', 'object_id') \
+            .annotate(count=Count('id')) \
+            .values('resource_file', 'object_id') \
+            .order_by() \
+            .filter(count__gt=1)
+        if dup_resource_files:
+            total = dup_resource_files.count()
```

Comment:
I think this count is not the total number of duplicate files. Feels like the following should give the right count:
`total = sum([duplicate['count'] for duplicate in dup_resource_files])`

Reply:
right `total = dup_resource_files.count()` gives us the total number of bins with each bin containing ResourceFiles that are duplicated.
`total = sum([duplicate['count'] for duplicate in dup_resource_files])` gives us the total number of actual files.

I think either is a fine way to count. I'll change the output to make it more clear what we are iterating